docs/user/ppl/admin/catalog.rst

```diff
@@ -0,0 +1,88 @@
+.. highlight:: sh
+
+=================
+Catalog Settings
+=================
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 1
+
+Introduction
+============
+
+The concept of ``catalog`` is introduced to support the federation of SQL/PPL query engine to multiple data sources.
+This helps PPL users(plugins/customer) to leverage data from multiple data sources and derive correlation and insights.
```

Comment:
what does `plugins/customer` mean?

Reply:
I meant opensearch plugins and opensearch users. Let me remove it completely.